tests/test_json.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+# ==============================================================================
+"""Tests for JSON Dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+import tensorflow_io.json as json_io
+from tensorflow_io.core.python.ops import data_ops as data_ops # pylint: disable=wrong-import-position
+
+
+def test_json():
+  """test_json"""
+  feature_filename = os.path.join(
+      os.path.dirname(os.path.abspath(__file__)),
+      "test_json",
+      "feature.json")
+  label_filename = os.path.join(
+      os.path.dirname(os.path.abspath(__file__)),
+      "test_json",
+      "label.json")
+  d_train_feature = json_io.JSONDataset(
+      feature_filename,
+  )
+  d_train_label = json_io.JSONDataset(
+      label_filename,
+  )
+
+  d_train = data_ops.Dataset.zip((
```

Comment:
I think you could use Tensorflow's public API `tf.data.Dataset.zip` instead.

Reply:
Done